app/models/business_objective.rb

```diff
@@ -6,6 +6,8 @@ class BusinessObjective < ApplicationRecord
   has_many :profiles, through: :policies
   has_many :accounts, through: :policies
 
+  delegate :account_number, to: :account
```

Comment:
There's really no reason that we should store account and reference related objects by its UUID `id`. Eventually, I'd like to remove the accounts table and store `account_number` directly on our models that require it. No, not all our models are directly related to an account; it's only hosts, profiles, and policies.

Reply:
One more point I'd like to raise here: if we remove the accounts table altogether, the cyndi hosts view with the `account` field won't be nearly as problematic, and we can remove any references to `account_object`.

Reply:
Requiring every table to have the association to an account might be able to shorten the distance, e.g. for checking whether the object belongs to an account. It could be a nightmare have it synced correctly and kept coherent with connected models. Seems like your targeting to have some kind of a single table inheritance for an Account. Full removal of an Account object might seem premature. In some form it could be kept, e.g. when there would be another source/view for Account data (like Hosts within Cyndi).

Technically only three models need an Account association: Hosts, Policies, and Business Objectives. Other models have associations to their parent. Therefore, it does not seems to make sense to have `account_number` (or other type of an account identifier) to be on all models.

Reply:
I'm not suggesting to have it on all models, just the three that have direct account associations, `hosts`, `policies`, and `profiles` (business_objectives have an account through policy). In any case, we can debate removal of the accounts table later. This PR does not remove it :)

Reply:
Perhaps not directly related, but I just realized our accounts table does not currently validate that account_number is unique. This could potentially cause only subsets of an account's data to be returned if there are duplicate account objects in our DB sharing account_numbers.